install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -188,3 +188,24 @@ following:
 pods. Either set the export to be owned by the container's primary UID, or give
 your pod group based access using `*SuppplementalGroups*`. See
 link:pod_security_context.html[Volume Security] for more information.
+
+
+[[nfs-additional-config-and-troubleshooting]]
+
+== Additional Configuration and Troubleshooting
+Depending on what version of NFS is being used and how it is configured, there may be additional
+configuration steps needed for proper export and security mapping.  Below are some links and topics that
+may apply:
+
+`*NFSv4 mount incorrectly shows all files with ownership of nobody:nobody*`
+
+- Could be attributed to the ID mapping settings (/etc/idmapd.conf) on your NFS client and server, as NFSv4 utilizes ID mapping to ensure permissions are set properly on exported shares, if the domains of the client and server do not match then the permissions are mapped to nobody:nobody
+- For link:https://access.redhat.com/solutions/33455[more information] on how to diagnose and resolve this issue
```

Comment:
hm, i'm guessing linking to content behind a paywall will not be popular in our origin docs.  @adellape ?

Also this sentence is going to read "For _more information_ on how to diagnose and resolve this issue" which is a fragment.